backend/hitas/migrations/0047_auto_20230105_1357.py

```diff
@@ -0,0 +1,41 @@
+# Generated by Django 3.2.16 on 2023-01-05 11:57
```

Comment:
Please use a verbose name the migration :)
e.g.: `./manage.py makemigrations hitas --name change_on_delete_rules`

Reply:
This change is in the latest commit.